helpdesk-service/docker/k8s_deployment.yaml.template

```diff
@@ -40,5 +42,3 @@ spec:
               value: 'true'
             - name: APP_AUTHENTICATION_FILTER_ENABLED
               value: 'false'
-            - name: MONGO_IGNORESSL
```

Comment:
You should include two new environment variables to support app authentication: APP_AUTHENTICATION_API_KEYSTORE_FILE and APP_AUTHENTICATION_API_KEYSTORE_PASSWORD

Reply:
Done.